activitychart-basic.html

```diff
@@ -106,29 +106,57 @@
       },
 
       _showLabel: function (d, i) {
```

Comment:
even though this method is private, we might wanna use more meaningful names than d, i